packages/client/wallets/smart-wallet/src/blockchain/wallets/EVMSmartWallet.ts

```diff
@@ -112,4 +125,39 @@ export class EVMSmartWallet {
     public async nfts() {
         return this.crossmintService.fetchNFTs(this.address, this.chain);
     }
+
+    public async sendTransaction<
+        const TAbi extends Abi | readonly unknown[],
+        TFunctionName extends ContractFunctionName<TAbi, "nonpayable" | "payable"> = ContractFunctionName<
+            TAbi,
+            "nonpayable" | "payable"
+        >,
+        TArgs extends ContractFunctionArgs<TAbi, "nonpayable" | "payable", TFunctionName> = ContractFunctionArgs<
+            TAbi,
+            "nonpayable" | "payable",
+            TFunctionName
+        >
+    >({
+        address,
+        abi,
+        functionName,
+        args,
+        resendCallback,
```

Comment:
This allows a dev using the SDK to e.g. make it so a popup appears that says "We need a new signature from your passkey" when we need a new signature to send a transaction. This may become less useful when we implement session keys but nice to have